sdk/compiler/script-service/server/src/main/scala/com/digitalasset/daml/lf/script/Context.scala

```diff
@@ -171,7 +177,13 @@ class Context(
           this.packageId = homePackageId
           this.allSignatures = extSignatures.updated(
             packageId,
-            Ast.PackageSignature(Map.empty, extSignatures.keySet, languageVersion, packageMetadata),
+            Ast.PackageSignature(
+              Map.empty,
+              extSignatures.keySet,
+              languageVersion,
+              packageMetadata,
+              Left("com.digitalasset.daml.lf.Context"),
```

Comment:
```suggestion
              Left("com.digitalasset.daml.lf.script.Context"),
```

Reply:
look at the first (non comment line of the file).
```scala 
package com.digitalasset.daml.lf
package script
```

see [scala doc](https://www.scala-lang.org/files/archive/spec/2.13/09-top-level-definitions.html#packages) for more details.